Source/WebCore/html/HTMLPlugInElement.cpp

```diff
@@ -374,26 +374,6 @@ bool HTMLPlugInElement::requestObject(const String& relativeURL, const String& m
     return true;
 }
 
-bool HTMLPlugInElement::setReplacement(PluginUnavailabilityReason reason, const String& unavailabilityDescription)
-{
-    Ref protectedThis { *this };
-    {
-        CheckedPtr renderer = dynamicDowncast<RenderEmbeddedObject>(this->renderer());
-        if (!renderer)
-            return false;
-
-        if (reason == PluginUnavailabilityReason::UnsupportedPlugin)
-            document().addConsoleMessage(MessageSource::JS, MessageLevel::Warning, "Tried to use an unsupported plug-in."_s);
-
-        renderer->setPluginUnavailabilityReasonWithDescription(reason, unavailabilityDescription);
-    }
-    bool replacementIsObscured = isReplacementObscured();
```

Comment:
This seems to be the last non-test call for this method.

Reply:
Indeed. Also, the Internals test API is not invoked by any test, so we can also remove that.